tests/ovm_test.go

```diff
@@ -252,19 +243,23 @@ var _ = Describe("OfflineVirtualMachine", func() {
 		})
 
 		It("should recreate VM if it gets deleted", func() {
-			newOVM := newOfflineVirtualMachine(true)
-			// Delete the VM
-			Eventually(func() error {
-				return virtClient.VM(newOVM.Namespace).Delete(newOVM.Name, &v12.DeleteOptions{})
-			}, 120*time.Second, 1*time.Second).Should(Succeed())
+			newOVM := startOVM(newOfflineVirtualMachine(false))
+
+			currentVM, err := virtClient.VM(newOVM.Namespace).Get(newOVM.Name, v12.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			Expect(virtClient.VM(newOVM.Namespace).Delete(newOVM.Name, &v12.DeleteOptions{})).To(Succeed())
 
 			Eventually(func() bool {
-				_, err := virtClient.VM(newOVM.Namespace).Get(newOVM.Name, v12.GetOptions{})
+				vm, err := virtClient.VM(newOVM.Namespace).Get(newOVM.Name, v12.GetOptions{})
 				if errors.IsNotFound(err) {
 					return false
 				}
-				return true
-			}, 120*time.Second, 1*time.Second).Should(BeTrue())
+				if vm.UID != currentVM.UID {
+					return true
+				}
+				return false
+			}, 240*time.Second, 1*time.Second).Should(BeTrue())
```

Comment:
Would you mind adding a fail message explaining what happened? Failed test result will simply say "False != True" otherwise